sys/dev/ahci/ahci_pci.c

```diff
@@ -664,6 +664,8 @@ static driver_t ahci_driver = {
         sizeof(struct ahci_controller)
 };
 DRIVER_MODULE(ahci, pci, ahci_driver, ahci_devclass, NULL, NULL);
+MODULE_PNP_INFO("U32:device;U8:rev;D:#", pci, ahci, ahci_ids,
```

Comment:
I think this needs to be "W32:vendor/device;" We can leave out the U8:rev; stuff since it won't change which driver is loaded.

Reply:
Yeah, I've excluded U8:rev and later portion and changed it to W32